### ISSUE TITLE
Add minimum system requirements for tachiyomij2k app 

### ISSUE THREAD

**Issue by lostb053** [tachiyomiorg/website#385]
Well i downloaded app but it won't support chhod I've old Android version.
So i think it would be good if i could see minimum system requirements
Sorry if it's there but i wasn't able to see

**Comment by Soitora** [comment]
You're right, there's no minimum system requirements for any fork I believe.

**Comment by lostb053** [comment]
yeah
recently i download tachiyomij2k app just to know that it ain't compatible
with my phone
